### PR TITLE
Add documentation for configurable navigation nodes

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Add documentation for configurable navigation nodes

**Related issue(s)**

### CODE REVIEW

File: docs/console/05-01-console.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ This table lists the configurable parameters, their descriptions, and default va
 | **cluster.headerLogoUrl** | Defines the address of the logo image that shows in the Console UI navigation header. | `assets/logo.svg` |
 | **cluster.faviconUrl** | Defines the icon that shows in the address bar of a browser. | `favicon.ico` |
 | **cluster.headerTitle** | Defines an additional title that shows next to the logo image in the Console UI navigation header. | None |
+| **cluster.disabledNavigationNodes** | An array of categories or specific nodes that you want to hide in the Console UI navigation. To hide a category, make sure the array includes **categoryLabel** for cluster-wide views. For the namespace-related views, add the **namespace** prefix to the label: **namespace.categoryLabel**. To hide a specific node, make sure the array includes **categoryLabel.nodeLabel** or **namespace.categoryLabel.nodeLabel**. For all labels, use lowercase and don't use any spaces or dashes. | [] |
```
Comment (klaudiagrz) [inline docs/console/05-01-console.md]:
```suggestion
| **cluster.disabledNavigationNodes** | An array of categories or specific nodes that you want to hide in the Console UI navigation. To hide a category, make sure the array includes **categoryLabel** for cluster-wide views. For the Namespace-related views, add the **namespace** prefix to the **namespace.categoryLabel** label. To hide a specific node, make sure the array includes **categoryLabel.nodeLabel** or **namespace.categoryLabel.nodeLabel**. For all labels, use lowercase and don't use any spaces or dashes. | [] |
```

Comment (kazydek) [reply docs/console/05-01-console.md]: And can we start the description with a verb? what does `[]` denote? We normally put values in code here.

Comment (klaudiagrz) [reply docs/console/05-01-console.md]: I guess `[]` stands for an array? But I agree with the verb part of the comment 👍

File: resources/core/charts/console/values.yaml [hunk 2]
```diff
@@ -34,3 +34,4 @@ cluster:
   faviconUrl: 'favicon.ico'
   # Provide custom title for the UI application that will be shown in the header component
   headerTitle: ''
+  disabledNavigationNodes: []
```
Comment (kwiatekus) [inline resources/core/charts/console/values.yaml]: I added string values to the values.yam here as follows:
```
disabledNavigationNodes: [
    "namespace.operation.secrets",
    "settings",
    "foo.bar"
  ]
```

which generated the following snippet in the config-map:
```
disabledNavigationNodes: '[namespace.operation.secrets settings foo.bar]'
```

This was not properly interpreted by console and the nodes were not hidden

Comment (kwiatekus) [reply resources/core/charts/console/values.yaml]: Fix proposed : https://github.com/kyma-project/console/pull/995

File: docs/console/05-01-console.md [hunk 3]
```diff
@@ -20,3 +20,4 @@ This table lists the configurable parameters, their descriptions, and default va
 | **cluster.headerLogoUrl** | Defines the address of the logo image that shows in the Console UI navigation header. | `assets/logo.svg` |
 | **cluster.faviconUrl** | Defines the icon that shows in the address bar of a browser. | `favicon.ico` |
 | **cluster.headerTitle** | Defines an additional title that shows next to the logo image in the Console UI navigation header. | None |
+| **cluster.disabledNavigationNodes** | Defines a list of categories or specific nodes in the navigation that you want to hide in the Console UI navigation. To hide all navigation nodes from a category, make sure the list includes the **categoryLabel**. To hide a specific navigation node, use both **categoryLabel** and **nodeLabel** separated with a `.`  For the Namespace-related views, add the **namespace** prefix to the list entry. For example, `namespace.operation.secrets` would hide **Secrets** navigation node within the **Operation** category for all namespaces. For all labels, use lowercase and don't use any spaces or dashes. | `''` |
```
Comment (kazydek) [inline docs/console/05-01-console.md]:
```suggestion
| **cluster.disabledNavigationNodes** | Defines a list of categories or specific nodes in the navigation that you want to hide in the Console UI navigation. To hide all navigation nodes from a category, make sure the list includes **categoryLabel**. To hide a specific navigation node, use both **categoryLabel** and **nodeLabel** separated with a period (`.`). For the Namespace-related views, add the `namespace` prefix to the list entry. For example, `namespace.operation.secrets` would hide the **Secrets** navigation node within the **Operation** category for all Namespaces. For all labels, use lowercase and don't use any spaces or dashes. | `''` |
```

**Pull request merged**